### PR TITLE
SPM-21 - Add Fragment logs data and pagination

### PR DESCRIPTION
**Pull request by lgobbi-atix**
- Add fragment logs information from node
- Add pagination and searching functionality for fragments

### CODE REVIEW

File: electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx [hunk 1]
```diff
@@ -1,65 +1,146 @@
...
+    if (currentPage > 1) setCurrentPage(currentPage - 1);
+  };
+
+  useEffect(() => {
+    fetchData(nodeAddress);
+  }, [nodeAddress]);
+
+  useEffect(() => {
+    if (fragmentLogs && fragmentLogs.length > 0) {
+      setTotalPages(Math.ceil(fragmentLogs.length / PAGE_SIZE))
```
Comment (lgobbi-atix) [inline electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx]:
```suggestion
      setTotalPages(Math.ceil(fragmentLogs.length / PAGE_SIZE));
```

File: electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx [hunk 2]
```diff
@@ -1,65 +1,146 @@
...
+          onClick={fetchData}
+        />
+      </div>
+      {fragmentLogs && fragmentLogs.length > 0 ? (
+        <TableFragments
+          fragments={fragmentLogs.slice(
+            beginningArray,
+            fragmentLogs.length - beginningArray < PAGE_SIZE
+              ? fragmentLogs.length
+              : PAGE_SIZE
```
Comment (lgobbi-atix) [inline electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx]:
```suggestion
              : beginningArray + PAGE_SIZE
```

File: electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx [hunk 3]
```diff
@@ -1,65 +1,146 @@
...
+
+  const [fragmentLogs, setFragmentLogs] = useState();
+  const [inputFragmentId, setInputFragmentId] = useState();
+  const [errorMessage, setErrorMessage] = useState(
+    getMessage('report.fragments.missing')
+  );
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+
+  const fetchData = async () => {
```
Comment (jmendiola222) [inline electron-app/src/components/molecules/ReportTool/FragmentLogs/FragmentLogs.jsx:22]: No necessary to this PoC, but when component reach some complexity, is a good practice to divide view from logic. ;)

File: electron-app/src/components/molecules/ReportTool/FragmentLogs/_style.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+@import '../../../../css/general_styling';
+
+
+.ContainerApp {
+.tabla {
```
Comment (jmendiola222) [inline electron-app/src/components/molecules/ReportTool/FragmentLogs/_style.scss:5]: table. ;)

**Review by jmendiola222 — APPROVED** [review]
:100:

**Pull request merged**